### PR TITLE
update store-passwords-using-reversible-encryption.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2411

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/security-policy-settings/store-passwords-using-reversible-encryption.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ Enabling this policy setting allows the operating system to store passwords in a
 
 Disable the **Store password using reversible encryption** policy setting.
 
+>[!Note]
+> After disabling the policy settings, only the new passwords will be forced to be stored using one-way encryption. Existing passwords will be stored using reversible encryption until the password is changed.
```
Comment (nenonix) [inline windows/security/threat-protection/security-policy-settings/store-passwords-using-reversible-encryption.md]:
```suggestion
> When policy settings are disabled, only new passwords will be stored using one-way encryption by default. Existing passwords will be stored using reversible encryption until they are changed.
```

File: windows/security/threat-protection/security-policy-settings/store-passwords-using-reversible-encryption.md [hunk 2]
```diff
@@ -69,6 +69,10 @@ Enabling this policy setting allows the operating system to store passwords in a
...
+
```
Comment (nenonix) [inline windows/security/threat-protection/security-policy-settings/store-passwords-using-reversible-encryption.md]:
```suggestion
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix ,
your recommendations were applied, thank you!

**Comment by mypil** [comment]
@Justinha @brianlic-msft - Can you please assist to check, approve and merge the content of this PR based on issue #2411?

Thank you.

**Pull request merged**